sections/03-methods.md

```diff
@@ -0,0 +1,65 @@
+
+Art's role in a practice/research assemblage for me, then, is to modulate theorization with the weird, serendipitous, and distinctly material resonances of creative action. For example, in the process of exposing the "inner vision" of a computational algorithm as light and movement, such as in _In Passing_, my videos enabled me to theorize computation as a material and embodied force--as more than simply a representation of the world, but also as a body that affects other bodies _in_ the world.
+
+#### Agonism and Antagonism
+
+Barry and Born's third mode, the one that they find most potent, carries with it an appealing methodological politics, which they argue can produce "more radical shifts in knowledge practices, shifts that are epistemic and/or ontological" (13). I have already argued for adjacency and drift as personal metamethods that aim to produce a degree of ontological uncertainty when practising in multiple disciplines. And while I am sympathetic to the possibility of epistemic and methodological radicalism (particularly in the realm of design practice, which so often carries the weight of consumerist and industrial values in its formalisms), I prefer to focus here on local consequences for my particular practice-based assemblage of videos, sounds, and software.
+
+It is important to note that, for me, working in multiple disciplines is not always straightforwardly generative, and my focus on certain long-reaching potentialities of software creativity often forecloses the experience of more immediately connections. Working in one discipline can often produce strange resonances and modulations, some of which amplify while others attenuate my work in the other disciplines. In particular, while my software development practice has the ability to elicit new kinds of creative or theoretical constructs by affording or limiting certain creative avenues, programming tends as often as not to disrupt and antagonize the work of making videos or new compositions.
```

Comment:
immediately -> immediate

This creates huge issues for the role of the "audience" of an artwork. Who is such a thing, and what relationship can/do they bear to the creator? Does this not inherently suggest that no audience that isn't personally involved with the creator is possible? Presumably the destruction of the primacy of "disciplines" also destroys the primacy of any such concept as an "audience".